question_answering_service/question_answering.py

```diff
@@ -14,26 +14,25 @@
 def generate_answer():
     try:
         question = request.args.get('question')
-
         prompt = construct_prompt(question)
-
         response = openai.Completion.create(
             prompt=prompt,
             **COMPLETIONS_API_PARAMS
         )
-
         answer = response["choices"][0]["text"].strip(" \n")
-
         return answer
 
     except Exception as e:
         logging.error("Error generating answer: " + str(e), exc_info=True)
-        return ""
+        return "", 500
```

Comment:
Shall we send a error msg also

Reply:
Addressed in [a12c2db](https://github.com/JayaniH/openai-usecases/pull/1/commits/a12c2dbc9cf69caf2f70c9968b50099a34c52547)